### PR TITLE
Minecart camera pan

### PR DESCRIPTION
**Pull request by joshlory**
Request from Joel, and follow up to PR https://github.com/code-dot-org/craft/pull/92.

This is the level before the 20x20 free-play world.  Include a bit of horizontal camera pan during the minecart animation, so it's clear the camera can move in the next level.

adventurer-13-pan-actual-speed

### CODE REVIEW

File: src/js/game/LevelMVC/LevelView.js [hunk 1]
```diff
@@ -546,6 +546,8 @@ module.exports = class LevelView {
     position = [3, 2];
 
     animation = this.playLevelEndAnimation(position, facing, isOnBlock, completionHandler, false);
+    this.game.world.setBounds(0, 0, this.controller.levelModel.planeWidth * 40, this.controller.levelModel.planeHeight * 40);
```
Comment (Hamms) [inline src/js/game/LevelMVC/LevelView.js]: should these bounds be set elsewhere? What happens if the player tries to move off the edge of the screen before and after these are set?

Comment (joshlory) [reply src/js/game/LevelMVC/LevelView.js]: Good catch — right now they can move off screen during the puzzle.  I will fix!

When we set the bounds for the cut scene animation, the user can no longer control the player so I think it's ok.

Comment (joshlory) [reply src/js/game/LevelMVC/LevelView.js]: Note that `this.game.world.setBounds` only constrains the camera.  Sprites don't take the bounds into account when moving.

**Comment by joshlory** [comment]
@Hamms PTAL!

**Pull request merged**